frontend/dockerfile/instructions/commands_runmount.go

```diff
@@ -151,6 +160,25 @@ func parseMount(value string) (*Mount, error) {
 		}
 
 		value := parts[1]
+		// possible env var, defer for later
+		matched, err := regexp.MatchString(`\$.`, value)
+		if err != nil {
+			return nil, err
+		}
+		if matched {
```

Comment:
Not all variables are allowed to be expended (atm). From/type are not allowed afaics. Add helpful errors to these cases.

Reply:
Added message for `from`. Regarding `type` - let's keep discussing in the other comment